express-post-json/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const app = express();
+const nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  var gradesArray = [];
+  for (var key in grades) {
+    gradesArray.push(grades[key]);
+  }
+  res.json(gradesArray);
+});
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
+
+const data = express.json();
+app.use(data);
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  res.send(req.body);
+  res.status(201);
```

Comment:
These should be reversed so that the status code is `201` instead of `200`. Check your response.

Reply:
![Screen Shot 2021-03-16 at 3 51 26 PM](https://user-images.githubusercontent.com/74885918/111390313-c59fad00-866f-11eb-8d69-8a9dcd379f6f.png)